.flake8

```diff
@@ -19,6 +19,7 @@ per-file-ignores =
     tools/infer/text/parallel/predict_system.py:E402
     tools/infer/text/predict_system.py:E402
     tools/infer/text/predict_rec.py:E402
+    tools/dataset_converters/convert.py:F401,F403
```

Comment:
F401 is enough?

Reply:
Yes, currently F401 is enough. But in convert.py and */builder.py we use `eval(class_name)`. In some */builder.py, we use `from module import *`. So I add F403 to avoid flake8 check failure on convert.py if the import way changes in future.